kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -962,10 +962,18 @@ messages:
       if liClient_cmd = BP_REQ_GET
       {
          oWhat = Nth(client_msg,2);
-         Send(self,@UserGet,#what=oWhat);
+         Send(self,@UserGet,#what=oWhat,#number=$);
```

Comment:
This change isn't necessary.

Reply:
Removed this change as it is not needed.